news/thr_unthr.rst

```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* Added ``xth`` (x-threadable) and ``xunth`` (x-unthreadable) aliases to force command running in thread or not.
```

Comment:
It is better to remove threading/unthreading behaviour at all. The main changing factor when running shell is capturing/non-capturing . Threading should be left to the shell to decide. Is there any difference in behaviour when running threaded or unthreaded ?

Reply:
OK, now I understand the case where one wants capture explicitly. 

3. We can add it if needed later. As you have mentioned anything that is specified with the surrounding `$()`